jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -278,7 +278,7 @@ else if (filled == -1 && getEndPoint().isOutputShutdown())
                 }
                 else if (filled == 0)
                 {
-                    fillInterested();
+                    tryFillInterested();
```

Comment:
We need to review all other calls to `fillInterested` as any of them can be register-bombed by some random thread doing a read